.github/workflows/tests.yml

```diff
@@ -1,29 +1,39 @@
 name: Tests
```

Comment:
Rename workflow to "Test Definitions" and file to `test_definitions.yml`

Reply:
Done both.